enginetest/queries/stats_queries.go

```diff
@@ -32,7 +32,8 @@ var StatisticsQueries = []ScriptTest{
 		},
 		Assertions: []ScriptTestAssertion{
 			{
-				Query: "SELECT * FROM information_schema.column_statistics",
+				SkipResultCheckOnServerEngine: true, // the types of mcvCounts and mcvs are not identified over the wire result
```

Comment:
Any more details on this one? Is `stats.Bucket` not being converted to JSON for the comparison? Or is it one of the `int64` values that's tripping things up?

Reply:
Ah yes, my guess on this was that we cannot detect specific type for `nil` value from result received from `go-sql-driver`. It fails from:
`"mcv_counts":[]uint64(nil), "mcvs":[][]interface {}{},`
 vs 
`"mcv_counts":interface{}(nil), "mcvs":[]interface {}{},`